scripts/generate.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+# (C) 2023 GoodData Corporation
+set -e
+shopt -s extglob
+
+content_dir=versioned_docs
+# Name of the remote hosting the main repo (gitlab.com/gooddata/gdc-tiger-docs)
+remote_name=${1:-origin}
+# target branch where changes will be applied (master, rel/0.7, ...)
+target_branch=${2:-master}
+# if set to "keep_master", the "docs" from master will be preserved (only for preview!)
+keep_master=$3
+
+echo "Validating target branch '$target_branch'"
+case "$target_branch" in
+    master)
+        current_section=""
+        src_dir="."
+        ;;
+    rel/@(+([[:digit:]]).+([[:digit:]])|cloud))
+        current_section="${target_branch#rel/}"
+        src_dir="latest"
+        ;;
+    *)
+        echo "Unsupported target branch '$target_branch'"
+        exit 1
+esac
+
+REPO_ROOT="$(cd "$(dirname "$0")/.." && pwd)"
+
+# TODO: update when we move hugo to repo root
+pushd "$REPO_ROOT/docs"
+
+rm -rf ${content_dir:?}/*
```

Comment:
```suggestion
rm -rf ${content_dir:?Empty content_dir variable}/*
```

Reply:
The `content_dir` is hardcoded on the line:6, so it should never be empty. Or is there such a scenario?